src/account-info/AccountDetailsScreen.js

```diff
@@ -17,9 +18,15 @@ const styles = StyleSheet.create({
   },
 });
 
+type SelectorProps = $ReadOnly<{|
+  user: UserOrBot,
```

Comment:
There are a few spots like this where the type changes are really correcting something that was wrong, and not just filling in something that was missing.

A piece of workflow for this sort of type-refactoring that I neglected to describe earlier, but makes a substantial difference in reading and reviewing the changes:
* For each correction like this one, pull it out as a separate commit, along with the follow-on corrections it requires. (For example, this file plus `AccountDetails.js` would I think be one commit.)
* Then the remaining changes, wherever they're all (a) boring and also (b) basically the same as each other, can be handy to squash together as one commit.

The major thing that does is make it much easier as the reader to trace the follow-on effects of each change, because there are a lot fewer files to look through at a time.

I can show you some Git features to help make it straightforward to edit your existing branch into this form.

Probably also a productive step: let's move the bulk of these type fixes into an independent PR, which we can take care of after merging this PR with the stream-settings-defaults feature. The one piece of it that's required for this PR is the one in `StreamScreen.js`.